### ISSUE TITLE
Compiling the core (BB #729)

### ISSUE THREAD

**Issue by OregonAsari** [OregonCore/OregonCore#729]
This issue was migrated from bitbucket.
**Original Reporter:** realyzee
**Original Date:** 02.11.2014 09:09:05 GMT+0000
**Original Priority:** blocker
**Original Type:** bug
**Original State:** resolved
**Direct Link:** https://bitbucket.org/oregon/oregoncore/issues/729

<hr>

There's an error compiling the newest core.

**Comment by OregonAsari** [comment]
This comment was migrated from bitbucket.
**Original User:** realyzee
**Original Date:** 02.11.2014 09:09:49 GMT+0000
**Direct Link:** https://bitbucket.org/oregon/oregoncore/issues/729/#comment-13365729

<hr>

**Comment by OregonAsari** [comment]
This comment was migrated from bitbucket.
**Original User:** realyzee
**Original Date:** 02.11.2014 09:17:47 GMT+0000
**Direct Link:** https://bitbucket.org/oregon/oregoncore/issues/729/#comment-13365803

<hr>

Fixed it with:

```
#!c++

if (fread(&referencedNode, sizeof(uint32), 1, tf) &&
                        !iLoadedSpawns.count(referencedNode))
                    {
                        ERROR_LOG("Trying to unload non-referenced model '%s' (ID:%u)", spawn.name.c_str(), spawn.ID);
                    }
```

**Comment by OregonAsari** [comment]
This comment was migrated from bitbucket.
**Original User:** realyzee
**Original Date:** 02.11.2014 09:19:00 GMT+0000
**Direct Link:** https://bitbucket.org/oregon/oregoncore/issues/729/#comment-13365826

<hr>

Another one :/

**Comment by OregonAsari** [comment]
This comment was migrated from bitbucket.
**Original User:** Zaffy
**Original Date:** 02.11.2014 10:56:32 GMT+0000
**Direct Link:** https://bitbucket.org/oregon/oregoncore/issues/729/#comment-13366718

<hr>

dont worry, will be fixed soon

**Comment by OregonAsari** [comment]
This comment was migrated from bitbucket.
**Original User:** Zaffy
**Original Date:** 02.11.2014 11:14:47 GMT+0000
**Direct Link:** https://bitbucket.org/oregon/oregoncore/issues/729/#comment-13366883

<hr>

Fix #729

→ <<cset c475725528f6>>
